src/avram/save_operation.cr

```diff
@@ -228,11 +228,6 @@ abstract class Avram::SaveOperation(T)
     save_status == SaveStatus::SaveFailed
   end
 
-  # :nodoc:
-  macro fillable(*args)
```

Comment:
This hasn't been a thing for a while, and I don't think it's a common enough term for people to accidentally use.